spec/controllers/api/v1/orders_controller_spec.rb

```diff
@@ -154,6 +154,13 @@
         expect(parsed_body['designations'].count).to eq(1)
         expect(parsed_body['meta']['total_pages']).to eql(1)
       end
+
+      it "should be able to fetch designations without their associations" do
+        get :index, shallow: true
+        expect(response.status).to eq(200)
+        expect(parsed_body.keys).to eq(['meta', 'designations'])
```

Comment:
Not sure if we need to sort the keys here or not... can't remember if this is deterministic. If you get random failures, then probably the ordering is important.